test/e2e/policies.test-suite/dynamic-pools/n4c16/test03-rebalancing/code.var.sh

```diff
@@ -36,41 +48,44 @@ verify-metrics-has-line 'containers="pod2:pod2c0".*dynamicPool="pool2",dynamicPo
 vm-command "nohup kubectl exec pod1 -- /bin/sh -c 'gzip </dev/zero >/dev/null' </dev/null >&/dev/null &"
 vm-command "nohup kubectl exec pod1 -- /bin/sh -c 'gzip </dev/zero >/dev/null' </dev/null >&/dev/null &"
 # Wait at least one rebalancing round and print CPU pinning.
-sleep 10
+sleep 2
 report allowed
 # Now "pool1" has 200% CPU load, "shared" and "pool2" have 0%.
 # Verify that the number of CPUs in pool1 is the largest.
-verify 'len(cpus["pod1c0"]) > len(cpus["pod0c0"])'
-verify 'len(cpus["pod1c0"]) > len(cpus["pod2c0"])'
-verify 'len(cpus["pod0c0"]) + len(cpus["pod1c0"]) + len(cpus["pod2c0"]) == 14'
+retry 10 verify \
+       'len(cpus["pod1c0"]) > len(cpus["pod0c0"])' \
+       'len(cpus["pod1c0"]) > len(cpus["pod2c0"])' \
+       'len(cpus["pod0c0"]) + len(cpus["pod1c0"]) + len(cpus["pod2c0"]) == 14'
 
 # Remove CPU load from pool1 and put 100% CPU load to pool2.
 vm-command "pkill gzip"
 vm-command "nohup kubectl exec pod2 -- /bin/sh -c 'gzip </dev/zero >/dev/null' </dev/null >&/dev/null &"
 # Wait at least one rebalancing round and print CPU pinning.
-sleep 10
+sleep 2
 report allowed
 # Verify that the number of CPUs in pool2 is the largest.
-verify 'len(cpus["pod2c0"]) > len(cpus["pod0c0"])'
-verify 'len(cpus["pod2c0"]) > len(cpus["pod1c0"])'
-verify 'len(cpus["pod0c0"]) + len(cpus["pod1c0"]) + len(cpus["pod2c0"]) == 14'
+retry 10 verify \
```

Comment:
@askervin @marquiz #1067 adds an implicit unparametrized builtin retry to `verify`. We need to chose whether we go with that, and make it configurable if needed, or with this.

Reply:
I'd rather change this into `verify --retry 10 ...` on top of PR #1069.

Reply:
I agree and suggested an alternative in #1067

Reply:
> I'd rather change this into `verify --retry 10 ...` on top of PR #1069.

OK, if we need 10 retries here then maybe #1069 would be the way to go